### ISSUE TITLE
bad argument #2 to 'narrow' when training own dataset

### ISSUE THREAD

**Issue by Aggr0vatE** [facebookresearch/deepmask#83]
Hi,

I am currently training my own dataset and am getting an error during conversion. I was able to successfuly train my dataset before when I had "iscrowd" on 0 and the segmentation to polygons, but now that I changed iscrowd to 1 and put in RLE numbers, I am getting the following error:

`OBRAs-Mac-Pro:test obra$ sudo th train.lua -maxepoch 50 -nthreads 1 -batch 18
batch	18	32	
-- ignore option rundir	
-- ignore option dm	
-- ignore option reload	
-- ignore option gpu	
-- ignore option datadir	
nthreads	1	2	
maxepoch	50	300	
| running in directory /test/exps/deepmask/exp,batch=18,maxepoch=50,nthreads=1	
| number of paramaters trunk: 15213312	
| number of paramaters mask branch: 1608768	
| number of paramaters score branch: 526337	
| number of paramaters total: 17348417	
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
converting: annotations
/Users/obra/torch/install/bin/luajit: /Users/obra/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] /Users/obra/torch/install/share/lua/5.1/coco/CocoApi.lua:65: bad argument #2 to 'narrow' (out of range at /Users/obra/torch/pkg/torch/lib/TH/generic/THTensor.c:359)
stack traceback:
	[C]: in function 'narrow'
	/Users/obra/torch/install/share/lua/5.1/coco/CocoApi.lua:65: in function '__init'
	/Users/obra/torch/install/share/lua/5.1/torch/init.lua:91: in function </Users/obra/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'TensorTable'
	/Users/obra/torch/install/share/lua/5.1/coco/CocoApi.lua:106: in function '__init'
	/Users/obra/torch/install/share/lua/5.1/torch/init.lua:91: in function </Users/obra/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'CocoSeg'
	/Users/obra/torch/install/share/lua/5.1/coco/CocoApi.lua:164: in function '__convert'
	/Users/obra/torch/install/share/lua/5.1/coco/CocoApi.lua:128: in function '__init'
	/Users/obra/torch/install/share/lua/5.1/torch/init.lua:91: in function </Users/obra/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'CocoApi'
	/test/DataSampler.lua:25: in function '__init'
	/Users/obra/torch/install/share/lua/5.1/torch/init.lua:91: in function </Users/obra/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataSampler'
	/test/DataLoader.lua:36: in function </test/DataLoader.lua:30>
	[C]: in function 'xpcall'
	/Users/obra/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/Users/obra/torch/install/share/lua/5.1/threads/queue.lua:65: in function </Users/obra/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/Users/obra/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	/Users/obra/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	/Users/obra/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	/Users/obra/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	/Users/obra/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	/test/DataLoader.lua:40: in function '__init'
	/Users/obra/torch/install/share/lua/5.1/torch/init.lua:91: in function </Users/obra/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataLoader'
	/test/DataLoader.lua:21: in function 'create'
	train.lua:101: in main chunk
	[C]: in function 'dofile'
	...obra/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x0107b461f0`

I am wondering if this is an issue with my json file or if there are some numbers that are incorrect in the orignal code. I looked through the files which are getting the errors and everything seemed fine. I need to finish this as soon as possible so I am going to be trying to figure this out as well.

Thanks for the help.

**Comment by nico00** [comment]
look at #72 or #71
